app/components/graphPanel/graph.panel.component.html

```diff
@@ -1,4 +1,10 @@
 <div class="graph-panel" id="graph-panel">
+  <!-- auto-detection box on commit graph -->
+  <div id="refresh-graph-alert" class="alert alert-warning" role="alert" tabindex="-1">
+    Commit graph changes detected!
+    <button id="refresh-graph" type="button" class="btn btn-warning" (click)="refreshGraph()">Refresh Graph</button>
+  </div>
+
```

Comment:
If the user clicks away from the modal without refreshing the graph, can the user refresh it another way?

Reply:
The button and alert box won't disappear until user refresh the graph.